<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGNDM</class>
 <widget class="QMainWindow" name="CGNDM">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(136, 138, 133);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_8">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>cancel download</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_10">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 87, 83);



</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>File</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="2">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
              </property>
              <property name="text">
               <string>Link</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
              </property>
              <property name="text">
               <string>directory</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="pushButton_2">
              <property name="styleSheet">
               <string notr="true">
.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
              </property>
              <property name="text">
               <string>Browser</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <layout class="QFormLayout" name="formLayout"/>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(211, 215, 207);
selection-background-color: rgb(136, 138, 133);</string>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">
.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
             </property>
             <property name="text">
              <string>       information sur download</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>youtube</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="2">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
            </property>
            <property name="text">
             <string>directory</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" rowspan="2">
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>115</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
            </property>
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_5">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>MP3</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButton_6">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>MP4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="2">
           <widget class="QComboBox" name="comboBox">
            <property name="styleSheet">
             <string notr="true">.QComboBox:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QComboBox:active {
	position:relative;
	top:1px;
}
</string>
            </property>
            <property name="minimumContentsLength">
             <number>2</number>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
            </property>
            <property name="text">
             <string>Browser</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" rowspan="2">
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
            </property>
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3" rowspan="2">
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QProgressBar" name="progressBar_3">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="styleSheet">
             <string notr="true">.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>playlist</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="4">
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="5" colspan="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="styleSheet">
           <string notr="true">.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="styleSheet">
           <string notr="true">.QPushButton:hover {
	background:linear-gradient(to bottom, #ccc2a6 5%, #eae0c2 100%);
	background-color:#ccc2a6;
}
.QPushButton:active {
	position:relative;
	top:1px;
}
</string>
          </property>
          <property name="text">
           <string>Browser</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
          </property>
          <property name="text">
           <string>directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QRadioButton" name="radioButton_8">
          <property name="text">
           <string>MP4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QRadioButton" name="radioButton_7">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>MP3</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>115</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QLabel  {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
  width: 160px;
  padding: 20px;
  overflow: hidden;
  border: none;
  font: normal 16px/1 &quot;Times New Roman&quot;, Times, serif;
 
	color: rgb(186, 189, 182);
  text-align: center;
  -o-text-overflow: ellipsis;
  text-overflow: ellipsis;
  background: #5a5a5a;
  -webkit-box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  box-shadow: 1px 1px 1px 0 rgba(0,0,0,0.3) ;
  text-shadow: 1px 1px 1px rgba(0,0,0,0.2) ;
}</string>
          </property>
          <property name="text">
           <string>Link</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5" colspan="2">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>show</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhjgj">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEXIT"/>
   </widget>
   <addaction name="menuhjgj"/>
  </widget>
  <action name="actionsource">
   <property name="text">
    <string>source</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>CGNDM</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
